include/openssl/ec.h

```diff
@@ -128,7 +128,7 @@ int EC_GROUP_copy(EC_GROUP *dst, const EC_GROUP *src);
  *  \param  src  source EC_GROUP object
  *  \return newly created EC_GROUP object or NULL in case of an error.
  */
-EC_GROUP *EC_GROUP_dup(const EC_GROUP *src);
+EC_GROUP *EC_GROUP_dup(const EC_GROUP *a);
```

Comment:
Please undo the rename, it isn't necessary, and considering `src` is used in `EC_group_copy`, we might as well keep the pattern.

Reply:
Done. The changed arg name was a copy&paste mistake.